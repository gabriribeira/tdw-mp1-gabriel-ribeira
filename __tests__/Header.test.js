import React from "react";
import { render } from "@testing-library/react";
import Header from "../components/header";

test("Header component snapshot", () => {
  const { container } = render(<Header />);
  expect(container).toMatchSnapshot();
});
