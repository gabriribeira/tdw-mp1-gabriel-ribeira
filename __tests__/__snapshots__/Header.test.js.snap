// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component snapshot 1`] = `
<div>
  <h2
    class="text-2xl md:text-4xl font-bold tracking-tight md:tracking-tighter leading-tight mb-20 mt-8"
  >
    <a
      class="hover:underline"
      href="/"
    >
      Blog do Gabriel Ribeira
    </a>
    .
  </h2>
</div>
`;
